### ISSUE TITLE
deprocate in favor of karma-phantomjs-launcher

### ISSUE THREAD

**Issue by robrich** [gskachkov/karma-phantomjs2-launcher#8]
Now that karma-phantomjs-launcher@1.0.0 includes phantomjs-prebuilt which includes phantomjs@2.1.1, do we need a separate karma-phantomjs2-launcher npm package?

**Comment by gskachkov** [comment]
Ohh, Cool. I missed that, karma-phantomjs2-launcher is temporary solution until phantomjs2  is not included in main branch, so we don't need karma-phantomjs2-launcher now.

**Comment by jesperronn** [comment]
@gskachkov could you please update README and changelog with proper deprecation notices?

**Comment by gskachkov** [comment]
@jesperronn yes,  I'll do. Do you know good examples how to do this? I'm quite newbie at npm. 
UPDATE: Ohh, I found command npm deprecate. Will do later today.

**Comment by robrich** [comment]
https://docs.npmjs.com/cli/deprecate is a good explanation for `npm deprecate karma-phantomjs2-launcher "use karma-phantomjs-launcher"` command.  Updating the README.md is just a git commit.

**Comment by robrich** [comment]
Probably good to deprecate `phantomjs2-ext` as well.

**Comment by matt-macleod** [comment]
`npm WARN deprecated karma-phantomjs2-launcher@0.5.0: Use karma-phantom-launcher@1.0.0 instead`
Just an FYI the deprecation notice tells you to install a non-existent node package `karma-phantom-launcher@1.0.0`. it should be `karma-phantomjs-launcher@1.0.0`

**Comment by gskachkov** [comment]
Oh, sorry. I've fixed.
